cdap-ui/app/cdap/components/PluginJSONCreator/constants.tsx

```diff
@@ -22,3 +22,5 @@ export const PluginTypes = Object.keys(GLOBALS.pluginTypeToLabel).filter(
 );
 
 export const WIDGET_TYPES = Object.keys(WIDGET_FACTORY);
+
+export const SPEC_VERSION = '1.5';
```

Comment:
Is this being used anywhere?

Reply:
Oops, I forgot to use it! 

I changed the code to use the constant at https://github.com/cdapio/cdap/pull/12214/files#diff-513ea481c1f91c43f68a702547a59be0

Reply:
Resolved!